src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -1349,7 +1412,7 @@ export default class ResultsViewOncoprint extends React.Component<
             if (clusteredHeatmapProfile === genesetHeatmapProfile) {
                 return this.genesetHeatmapTrackGroupIndex;
             } else {
-                const heatmapGroup = this.molecularProfileIdToHeatmapTracks[
+                const heatmapGroup = this.molecularProfileIdToAdditionalTracks[
```

Comment:
Same here,  if we are using `HeatmapTrack` to represent `AdditionalTrack`, maybe we should rename some of the variables.

Reply:
additionaltracks holds all of these, and because clusteredHeatmapProfile we know is heatmap, we know that this entry is a heatmap group. I'm not sure I understand the issue